create/remote-pipelines-tasks-resolvers.adoc

```diff
@@ -10,20 +10,22 @@ Pipelines and tasks are reusable blocks for your CI/CD processes. You can reuse
 
 In a pipeline run resource, you can specify a pipeline from an existing source. In a pipeline resource or a task run resource, you can specify a task from an existing source.
 
-In these cases, the _resolvers_ in {pipelines-title} retrieve the pipeline or task definition from the specified source at run time.
+`StepAction` objects are reusable actions that a single step within a task completes. When specifying a step, you can reference a `StepAction` object from an existing source.
+
+In these cases, the _resolvers_ in {pipelines-title} retrieve the pipeline, task, or `StepAction` definition from the specified source at run time.
 
 The following resolvers are available in a default installaton of {pipelines-title}:
 
-Hub resolver:: Retrieves a task or pipeline from the Pipelines Catalog available on {artifact-hub} or {tekton-hub}.
-Bundles resolver:: Retrieves a task or pipeline from a Tekton bundle, which is an OCI image available from any OCI repository, such as an OpenShift container repository.
-Cluster resolver:: Retrieves a task or pipeline that is already created on the same {OCP} cluster in a specific namespace.
+Hub resolver:: Retrieves a task, pipeline, or `StepAction` definition from the Pipelines Catalog available on {artifact-hub} or {tekton-hub}.
+Bundles resolver:: Retrieves a task, pipeline, or `StepAction` definition from a Tekton bundle, which is an OCI image available from any OCI repository, such as an OpenShift container repository.
+Cluster resolver:: Retrieves a task, pipeline, or `StepAction` definition that is already created on the same {OCP} cluster in a specific namespace.
 Git resolver:: Retrieves a task or pipeline binding from a Git repository. You must specify the repository, the branch, and the path.
 
-An {pipelines-shortname} installation includes a set of standard tasks that you can use in your pipelines. These tasks are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
+An {pipelines-shortname} installation includes a set of standard tasks and `StepAction` definitions that you can use in your pipelines. These tasks and `StepAction` definitions are located in the {pipelines-shortname} installation namespace, which is normally the `openshift-pipelines` namespace. You can use the cluster resolver to access the tasks.
```

Comment:
@vdemeester since shipped step actions will be unusable in 1.16, should we even mention them here?

Reply:
I discussed it with Savita annd there is an HTTP workaround that I will document.